external/leveldb/leveldb_tuning.ipynb

```diff
@@ -0,0 +1,767 @@
+      "  \"homogeneous_random_forest_regression_model_config.n_estimators\": 10,\n",
+      "  \"homogeneous_random_forest_regression_model_config.features_fraction_per_estimator\": 1,\n",
+      "  \"homogeneous_random_forest_regression_model_config.samples_fraction_per_estimator\": 0.9,\n",
+      "  \"homogeneous_random_forest_regression_model_config.regressor_implementation\": \"DecisionTreeRegressionModel\",\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.criterion\": \"mse\",\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.splitter\": \"best\",\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_depth\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_split\": 2,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_leaf\": 3,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_weight_fraction_leaf\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_features\": \"auto\",\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_leaf_nodes\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_impurity_decrease\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.ccp_alpha\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_to_fit\": 10,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.n_new_samples_before_refit\": 1,\n",
+      "  \"homogeneous_random_forest_regression_model_config.bootstrap\": 1,\n",
+      "  \"experiment_designer_config.utility_function_implementation\": \"ConfidenceBoundUtilityFunction\",\n",
+      "  \"experiment_designer_config.numeric_optimizer_implementation\": \"RandomSearchOptimizer\",\n",
+      "  \"experiment_designer_config.confidence_bound_utility_function_config.utility_function_name\": \"upper_confidence_bound_on_improvement\",\n",
+      "  \"experiment_designer_config.confidence_bound_utility_function_config.alpha\": 0.1,\n",
+      "  \"experiment_designer_config.random_search_optimizer_config.num_samples_per_iteration\": 1000,\n",
+      "  \"experiment_designer_config.fraction_random_suggestions\": 0.5,\n",
+      "  \"experiment_designer_config_fraction_random_suggestions\": 0.1\n",
+      "}.\n",
+      "10/08/2020 21:58:26 -   BayesianOptimizerFactory -    INFO - [BayesianOptimizerFactory.py:  40 -    create_local_optimizer() ] Creating a bayesian optimizer with config: {\n",
```

Comment:
This optimizer creation output seems repeated a number of times.  What's up with that?